exist-core/src/main/java/org/exist/http/servlets/HttpRequestWrapper.java

```diff
@@ -186,9 +186,6 @@ private void parseMultipartContent() {
         // Create a factory for disk-based file items
         final DiskFileItemFactory factory = new DiskFileItemFactory();
 
-        // Dizzzz: Wonder why this should be zero
-        factory.setSizeThreshold(0);
```

Comment:
duh...

Reply:
I am learning so much from your comments
This one was almost ancient from 2009 or so :)